cli/command/container/signals_windows.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux,!darwin
```

Comment:
Build tags are redundant here. `_windows` implicitly makes this only build on windows.

Reply:
Good point! I've repushed, removing the build tags here.